activesupport/test/claims_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'abstract_unit'
+
+class ClaimsTest < ActiveSupport::TestCase
+
+  def test_verify
+    claims = { pld: "data", for: "test", exp: "2021-12-31T18:30:00.000Z" }
+    assert_equal claims[:pld], ActiveSupport::Claims.verify!(claims, { for: "test" })
```

Comment:
One of the things I really don't like is adding hash braces where they're not needed. Let's remove them in the `verify!` call and other places.